python_modules/dagster/dagster/_core/storage/runs/base.py

```diff
@@ -373,6 +373,7 @@ def get_backfills(
         status: Optional[BulkActionStatus] = None,
         cursor: Optional[str] = None,
         limit: Optional[int] = None,
+        filters: Optional[BulkActionsFilter] = None,
```

Comment:
Should we consider changing the order of args?  And maybe (in a separate PR) switching callers from using status to using filters?

Reply:
yeah i think ideally filters would be before status, but there could be callsites that dont use kwargs right? `get_backfills(BulkActionStatus.COMPLETED)`

Reply:
I don't see any callsites that don't use kwargs.

Reply:
We would potentially worry about user code callsites, but `get_backfills` is not marked as `@public`, so I think you could add a changelog note and be fine about changing it.

Reply:
i'll stack a branch to do this

Reply:
branch for re-ordering https://github.com/dagster-io/dagster/pull/23773